### PR TITLE
Changed example email in the documentation

### PR DESCRIPTION
**Pull request by restuccia**
Changed the email used in the example to a non-existing one, to avoid spam

### CODE REVIEW

**Comment by geoffreyaldebert** [comment]
We are at the origins of this schema. I think we need to provide a valid adress for this one. If info does not fit, could we use schema@data.gouv.fr ? 
WDYT ?

**Comment by abulte** [comment]
@geoffreyaldebert well you just published it on the internet 🙈 so let's try it and what spam it brings up.

File: web/collections/_documentation/ajouter-un-schema.md [hunk 1]
```diff
@@ -35,5 +35,5 @@ Voici un exemple complet pour ajouter le schéma decp-dpa, au format Table Schem
 decp-dpa:
   url: https://github.com/etalab/schema-decp-dpa.git
   type: tableschema
-  email: info@data.gouv.fr
+  email: example@example.com
```
Comment (abulte) [inline web/collections/_documentation/ajouter-un-schema.md:38]:
```suggestion
  email: schema@data.gouv.fr
```

**Comment by geoffreyaldebert** [comment]
I think it's already everywhere on the internet but ok I edited it ;) => I merge PR

**Pull request merged**